docs/user-guide/end-user.md

```diff
@@ -36,6 +36,40 @@ The Overpass API takes a few seconds to respond, and after that you should get n
 point and polygon layers for the toilets of London! (nodes and ways in OpenStreetMap
 with the `amenity`=`toilet` tag on them)
 
+**Try to run a preset**
```

Comment:
Maybe refactor the doc to have title with `#`, `##`, `###` so we can have HTML anchor in links, very convenient.